src/containers/chat/chat.js

```diff
@@ -13,15 +16,21 @@ import ThemeContext from '../../context/theme-context';
 export const Chat = () => {
   const [iconsVisible, setIconsVisible] = useState(false);
   const [mailFormVisible, setMailFormVisible] = useState(false);
-  const { language, contacts } = useSelector((state) => ({
-    language: state.Language.language,
-    contacts: state.Contacts.contacts
+  const { language } = useSelector((state) => ({
+    language: state.Language.language
   }));
```

Comment:
shouldn't we get language from i18n?

Reply:
Yes, but this compnent refactors Volodumyr